semanticdb/scalac/library/src/main/scala/scala/meta/internal/semanticdb/scalac/SemanticdbPlugin.scala

```diff
@@ -13,7 +12,8 @@ class SemanticdbPlugin(val global: Global)
   val name = SemanticdbPlugin.name
   val description = SemanticdbPlugin.description
 
-  hijackReporter()
+  override val semanticdbReporter = hijackReporter()
```

Comment:
FYI the `override` here shouldn't be necessary, I think you can remove `val semanticdbReporter = None` in `ReporterOps` and it will still work

Reply:
We need to have a reference to pick it up in `ReporterOps`. Your proposal can be done if we change `val semanticdbReporter:Option[SemanticdbReporter] = None` to `def semanticdbReporter: Option[SemanticdbReporter]` in `ReporterOps`. By doing so it's not an override and it can be removed.